tests/test_defaults.py

```diff
@@ -0,0 +1,66 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2022, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations # isort:skip
+
+import pytest ; pytest
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+import subprocess
+from pathlib import Path
+
+# Bokeh imports
+from tests.support.defaults import collect_defaults, output_defaults
+
+#-----------------------------------------------------------------------------
+# Setup
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+class TestDefaults:
+    def test_defaults(self) -> None:
+        baseline = Path(__file__).parent / "baselines" / "defaults.json"
+
+        defaults = collect_defaults()
+        output_defaults(baseline, defaults)
+
+        status, out, _ = diff_baseline(baseline)
```

Comment:
What is the process to get test passing again on occasions where we do intentionally have to change a default value?

Reply:
The procedure is the same as in bokehjs' tests. The test generates a new baseline and overwrites the existing one in the repository. If there's a difference, the test fails and the diff is showed. The same diff can be obtained with `git diff`. Differences are reviewed and, if everything is in order, committed to the repository.